index.js

```diff
@@ -127,14 +127,15 @@ program
 program
   .command('serve [root]')
   .description('build then serve a website from a directory')
+  .option('-f, --force-reload', 'force a full reload of all site files when a file is changed')
   .option('-p, --port <port>', 'port for server to listen on (Default is 8080)')
   .option('--no-open', 'do not automatically open the site in browser')
   .action((root, options) => {
     const rootFolder = path.resolve(root || process.cwd());
     const logsFolder = path.join(rootFolder, '_markbind/logs');
     const outputFolder = path.join(rootFolder, '_site');
 
-    const site = new Site(rootFolder, outputFolder);
+    const site = new Site(rootFolder, outputFolder, options.forceReload);
```

Comment:
Seems weird for the `Site`'s constructor to accept this parameter (because this is a build configuration parameter, rather than a site configuration parameter).

Maybe we should pass `options.forceReload` when calling `site.rebuildAffectedSourceFiles()` instead?

Reply:
Yes, I tried that previously, but I ran into some issues because of the way delay() was implemented, it only accepts a function that takes single array of paths so I can't add a second parameter to it (see PR comments for more info).

Maybe I can set the variable on Site separately?

Reply:
> Maybe I can set the variable on Site separately?

I guess it is ok for now, might be worth a revisit in case there's more options in the future though.